pages/index.js

```diff
@@ -1,11 +1,14 @@
 import Head from "next/head";
 import styled from "styled-components";
 import LostItem from "../components/LostItem";
-import SearchbarLostItem from "../components/SearchbarLostItem";
 import FoundItem from "../components/LostItem";
 import { lostItems, foundItems } from "../components/Db";
+import { useState } from "react";
 
 export default function Home() {
+  const [search1, setSearch1] = useState("");
+  const [search2, setSearch2] = useState("");
```

Comment:
Your state and the corresponding setter function could have more descriptive names, like `[searchTerm, setSearchTerm]` for instance.